app.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express');
+var trucks = require('./trucks');
```

Comment:
Gahh, thanks for pointing this out. I noticed I still had the file in after my first push so I took it out, I should've looked for the reference